### PR TITLE
configuration_manual/nfs: Add note about actimeo and nolock with inde…

### PR DESCRIPTION
**Pull request by sirainen**
…x mountpoint

### CODE REVIEW

File: source/configuration_manual/nfs.rst [hunk 1]
```diff
@@ -30,8 +30,21 @@ NFS mount options
 * ``root_squash``: Dovecot doesn't care about this. Typically Dovecot doesn't
   store any root-owned files in NFS.
 
-* ``nolock``: This is possible to use either if ``lock_method=dotlock`` is
-  used, or as a slightly unsafe optimization (see below).
+* ``nolock``: This is possible to use as a slightly unsafe optimization.
```
Comment (cmouse) [inline source/configuration_manual/nfs.rst:33]: there is also local_lock=all option.

Comment (cmouse) [reply source/configuration_manual/nfs.rst:33]: local_lock=mechanism

Specifies whether to use local locking for any or both of the flock and the POSIX locking mechanisms. mechanism can be one of all, flock, posix, or none. This option is supported in kernels 2.6.37 and later.

The Linux NFS client provides a way to make locks local. This means, the applications can lock files, but such locks provide exclusion only against other applications running on the same client. Remote applications are not affected by these locks.

Comment (sirainen) [reply source/configuration_manual/nfs.rst]: Added. I think it's equivalent to nolock.

**Comment by cmouse** [comment]
few other things to consider, we could mention `relatime` or `noatime`, also we should mention `intr`.

**Comment by sirainen** [comment]
Ah, noatime definitely. But what should I say about intr? I don't think it's really related to Dovecot?

**Comment by sirainen** [comment]
Also:

       intr / nointr  This  option is provided for backward compatibility.  It
                      is ignored after kernel 2.6.25.

File: source/configuration_manual/nfs.rst [hunk 2]
```diff
@@ -27,11 +27,27 @@ NFS mount options
 * ``nordirplus``: Disable readdirplus operations, which aren't needed by
   Dovecot. They can also slow down some NFS servers.
 
+* ``noatime``: Disable updating atime automatically. Dovecot doesn't need this
+  and it may slow down NFS servers.
```
Comment (cmouse) [inline source/configuration_manual/nfs.rst:31]: I wonder if the work `automatically` is extra here.

**Review by cmouse — CHANGES_REQUESTED** [review]

**Review by cmouse — APPROVED** [review]

**Pull request merged**